src/tests/unit/fixtures/stubresponse.py

```diff
@@ -49,6 +49,10 @@ def iter_content(self, buffer_size=-1):
             return_list.append(part)
         return return_list
 
+    def iter_lines(self, **kwargs):
+        """Create iterated lines from the 'content' parameter."""
+        return self.content.splitlines()
```

Comment:
This should yield the results individually to match the expected functionality

Reply:
It shouldn't make one iota of difference functionally, but I will do so.